hibernate-core/src/test/java/org/hibernate/orm/test/intg/sqm/HqlTranslationNoFactoryTests.java

```diff
@@ -1,150 +0,0 @@
-/*
- * Hibernate, Relational Persistence for Idiomatic Java
- *
- * License: GNU Lesser General Public License (LGPL), version 2.1 or later
- * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
- */
-package org.hibernate.orm.test.intg.sqm;
-
-import java.util.Collections;
-import java.util.Optional;
-
-import org.hibernate.boot.spi.BootstrapContext;
-import org.hibernate.boot.spi.MetadataImplementor;
-import org.hibernate.engine.jdbc.spi.JdbcServices;
-import org.hibernate.engine.spi.SessionFactoryImplementor;
-import org.hibernate.internal.util.config.ConfigurationHelper;
-import org.hibernate.orm.test.jpa.JpaComplianceStub;
-import org.hibernate.metamodel.internal.JpaStaticMetaModelPopulationSetting;
-import org.hibernate.metamodel.model.domain.JpaMetamodel;
-import org.hibernate.metamodel.model.domain.internal.JpaMetamodelImpl;
-import org.hibernate.metamodel.MappingMetamodel;
-import org.hibernate.metamodel.spi.RuntimeModelCreationContext;
-import org.hibernate.query.criteria.ValueHandlingMode;
-import org.hibernate.query.hql.HqlTranslator;
-import org.hibernate.query.hql.internal.StandardHqlTranslator;
-import org.hibernate.query.hql.spi.SqmCreationOptions;
-import org.hibernate.query.internal.NamedObjectRepositoryImpl;
-import org.hibernate.query.named.NamedObjectRepository;
-import org.hibernate.query.spi.QueryEngine;
-import org.hibernate.query.sqm.spi.SqmCreationContext;
-import org.hibernate.query.sqm.tree.SqmStatement;
-import org.hibernate.type.spi.TypeConfiguration;
-
-import org.hibernate.testing.orm.domain.StandardDomainModel;
-import org.hibernate.testing.orm.junit.DomainModel;
-import org.hibernate.testing.orm.junit.DomainModelScope;
-import org.hibernate.testing.orm.junit.FailureExpected;
-import org.hibernate.testing.orm.junit.ServiceRegistry;
-import org.hibernate.testing.orm.junit.ServiceRegistryScope;
-import org.junit.jupiter.api.Test;
-
-/**
- * Tests making sure that HQL queries can fully be translated without a SessionFactory.
- *
- * todo (6.0) : have a way to directly load these translations into the {@link NamedObjectRepository}
- * 		directly.  For example Quarkus could store its build-time translations and directly here during boot-strap of the SF
- *
- * @author Steve Ebersole
- */
-@ServiceRegistry
-@DomainModel( standardModels = StandardDomainModel.RETAIL )
-public class HqlTranslationNoFactoryTests {
```

Comment:
Was the deletion of this file intentional?

Reply:
Yeah.  I added it to try to make sure that you could build the HQL translator without reference to a SessionFactory.  The intention was for Quarkus but they do not need this functionality